src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-image: url(images/star-active.svg);
+}
+
+.stars__star--inactive {
+  background-image: url(images/star.svg);
+}
```

Comment:
do this, but for each block (stars--0, stars--1...), and use `nth-child()` as i mentioned above